2996_reflection/reflection.md

```diff
@@ -3662,6 +3676,10 @@ Modify the definition of _manifestly constant-evaluated_ to leverage that of _pl
 * [#.#]{.pnum} the result of substitution into an atomic constraint expression to determine whether it is satisfied ([temp.constr.atomic]{.sref}), or
 * [#.#]{.pnum} the initializer for a variable that is usable in constant expressions or has constant initialization ([basic.start.static]{.sref}).
 
+::: addu
+[All plainly constant-evaluated expressions are manifestly constant-evaluated, but some manifestly constant-evaluated expressions (e.g., initializers that can require trial evaluations) are not plainly constant-evaluated.]{.note}
```

Comment:
I'd extend this to include the sentence you told me in the other comment. Something like:

> Such expressions will still be evaluated during translation, but unlike plainly constant-evaluated expressions, may be evaluated multiple times and there are no guarantees made about the order of their evaluation during translation.

?

Reply:
Great suggestion, thanks. Done.,